### DISCUSSION TITLE
Decrease the size of the beatmaps at profile page

### DISCUSSION THREAD

**Discussion by MeerukYang** [ppy/osu#30649] [Ideas] [answered]
In current version, the beatmaps shown at profile page has larger size than beatmap listing page.

![image](https://github.com/user-attachments/assets/97aef9f9-52ac-4107-b825-8e1841ef2cdb)
![image](https://github.com/user-attachments/assets/cec678f7-1af6-4c1e-9809-f9936f5a660b)

This makes the profile page looks bloated and weird when scrolled down.

And the size of it seems larger from the version of 2024.1009.1, as it used to show 2 in one row for me.

**Comment by Joehuu** [comment]
You might have changed the ui scale. At 1920x1080 @ 1x ui scale, beatmap listing shows 3 in one row and profile shows 2.

But yeah, the size should be consistent.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/30748
